### PR TITLE
feat(rust): change name like arguments value names across cli command

### PR DESCRIPTION
**Pull request by rjtch**
Related issue: https://github.com/build-trust/ockam/issues/5031

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

### CODE REVIEW

**Comment by davide-baldo** [comment]
Hi! thank you for the interest and time invested! If you need any help or have any questions, we are here to provide support. Please feel free to ask us **anything**.

PS: in order to merge you need to accept the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified

File: implementations/typescript/package-lock.json
Comment (adrianbenavides) [inline implementations/typescript/package-lock.json:1]: @rjtch I think you added this file by mistake. Could you remove it from the PR?

Comment (rjtch) [reply implementations/typescript/package-lock.json:1]: yeah it was a mistake, i take it off

**Comment by rjtch** [comment]
@davide-baldo  is my branch ok and ready?

**Review by adrianbenavides — DISMISSED** [review]
LGTM! Congratulations @rjtch on your first contribution to ockam! 🥳

**Review by adrianbenavides — APPROVED** [review]
LGTM! Congratulations @rjtch on your first contribution to ockam! 🥳

**Review by etorreborre — APPROVED** [review]

**Pull request merged**